### PR TITLE
Update iOS README.md

### PR DESCRIPTION
**Pull request by magicwave**

### CODE REVIEW

File: iOS/README.md [hunk 1]
```diff
@@ -1,3 +1,22 @@
 # ChatDemo-SwiftUI  
 
-Basic Chat application written with SwiftUI  
+Basic Chat application written with Ditto and SwiftUI/Combine  
+
+1. Clone this repo to a location on your machine and open in Xcode.
```
Comment (okdistribute) [inline iOS/README.md:5]: Is there anything required here for APP_ID/TOKEN with `Env.swift` for publishing to the app store? @kndoshn

Comment (kndoshn) [reply iOS/README.md:5]: Yes, I think we should hide the license token and the app ID like other apps: https://www.notion.so/getditto/Environment-Variables-78261e05a2b44a299ee388f06e9ff86a

@magicwave Please feel free to reach out if you have any questions.

Comment (magicwave) [reply iOS/README.md:5]: Ok, I'll implement the Env.swift pattern and update this PR

Comment (magicwave) [reply iOS/README.md]: Implemented Env.swift pattern and updated iOS README. 
(Note: rebased onto main and force pushed this PR branch)

**Review by okdistribute — APPROVED** [review]

**Pull request merged**